tests/testthat/test-fit_model.R

```diff
@@ -155,11 +155,11 @@ test_that("Bad params w/ failing fit issues warning and returns NA", {
     chains = 1,
     adapt_delta = 0.8,
     max_treedepth = 10,
-    iter_warmup = -100,
-    iter_sampling = 100
+    iter_warmup = -25,
```

Comment:
Is there a reason this is "-"25?

Reply:
Yeah it's displaying the diff weirdly but see the header comment on this test.